docs/notebooks/introduction_demo.ipynb

```diff
@@ -132,7 +125,105 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You'll notice that there are more parameters than we manually set. Parameters are created automatically by the nodes if needed. In particular functional nodes often create extra parameters for book keeping. In general the user should not need to worry about these extra parameters. They can access the ones of interest with the dictionary notation."
+    "You'll notice that there are more parameters than we manually set. Some of these are for internal bookkeeping. Parameters are created automatically by the nodes if needed. In general the user should not need to worry about these extra parameters. They can access the ones of interest with the dictionary notation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Function Nodes\n",
+    "\n",
+    "Sampling functions, such as those provided by numpy, are only one type of function that we might want to use to generate parameters. We might want to sample from other functions or apply a mathematical transform to multiple input parameters to compute a new parameter. For example consider the case of computing the `distmod` parameter from `redshift`. We can do this using the information about the cosmology, such as provided by astropy's `FlatLambdaCDM` class:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from astropy.cosmology import FlatLambdaCDM\n",
+    "\n",
+    "cosmo_obj = FlatLambdaCDM(H0=73.0, Om0=0.3)\n",
+    "redshifts = np.array([0.1, 0.2, 0.3])\n",
+    "distmods = cosmo_obj.distmod(redshifts).value\n",
+    "print(distmods)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Importantly, you can use a `FunctionNode` that takes input parameter(s) and produces output parameter(s) during the generation `FunctionNode` is a subclass of `ParameterizedNode` that wraps the functionality of collecting the inputs, running the computations, and storing the output. The user simply needs to provide the function node with the function it will use and the parameters. For example, TDAstro has a `DistModFromRedshift` class that wraps the previous operation.\n",
```

Comment:
```suggestion
    "Importantly, you can use a `FunctionNode` that takes input parameter(s) and produces output parameter(s) during the generation. `FunctionNode` is a subclass of `ParameterizedNode` that wraps the functionality of collecting the inputs, running the computations, and storing the output. The user simply needs to provide the function node with the function it will use and the parameters. For example, TDAstro has a `DistModFromRedshift` class that wraps the previous operation.\n",
```

Reply:
Hard to see, but adding a period after "generation"